### ISSUE TITLE
Citizen Manager keeps resetting unassigned citizens instead of leaving them unassigned.

### ISSUE THREAD

**Issue by Crit-Cee** [yairm210/Unciv#8744]
**Platform**
Operating system type, major version, architecture (64/32 bit).

Android 10, Galaxy Note 9.

**Version**
Where you got the game and the exact version number.

Google Play Store. Version 4.3.1

**Describe the bug**

Many times after conquering a city, I unassign citizens (with all specialists slots filled they sit idle producing only 1 production) to reduce the city population to keep the nation from becoming unhappy. Ever since the additions of more options to the citizen management I've noticed that idle citizens keep getting reassigned. This doesn't always happen, and I haven't been able to figure out what triggers it. Furthermore, the game seems to reassign the citizens, and then recalculate growth/starvation amount after that, making it impossible sometime to reduce the city population. This has happened in this case with Copenhagen. I tried turning on avoid growth, focus production, reset citizens then unassign again, etc. Nothing has worked.

**To Reproduce**

Click on a city, go to the city management screen. Unassign workers so there is starvation, all specialist slots are filled, and leave the rest idle. Exit screen and click next turn. Go back to same city management screen. Sometimes the citizens are reassigned. If this happens it always happens for that city, and the food amount stored is calculated after the citizens were reset by the manager. If it doesn't then it doesn't happen again for that city until either a building is built, tech is researched or social policy is acquired. For some reason those things reset unassigned citizens in all cities.

If game data is required, please add it here. From the menu, choose save game (or map), copy to clipboard, and paste here between the backtick blocks (or, if no such save is required, you may delete this):

I'm unable to add my save game file. I clicked copy saved game to clipboard and it gives a popup saying it was unable to sopy to clipboard. It works for new games, so my assumption is that the save game is too big to copy for my device.

<details><summary>Saved game</summary>

```

```
</details>

**Expected behavior**

When I unassign citizens it should leave them unassigned. Also, calculation of resources produced for production of buildings/units, food, etc, should have been first, before the manager auto assigns citizens.

**Screenshots**

![UncivCMbug1](https://user-images.githubusercontent.com/45017497/221288586-e4e028da-856e-43b4-929e-fc43e65a3b16.jpg)
![UncivCMbug2](https://user-images.githubusercontent.com/45017497/221288603-ecdc4063-85bc-445c-b5da-ead76185df45.jpg)

**Additional context**

An option to disable the citizen manager in the gameplay settings or part of the citizen manager drop down would be nice. So it would just not assign any new/idle citizens at all and new citizens would automatically be idle until the player assigns them.

**Comment by vegeta1k95** [comment]
Could it be that some of the tiles were blockaded by enemy units?

If enemy unit blocks a tile that is worked, city automatically reassigns citizens to account for that blocked tile (since blocked tiles do not produce yields).

**Comment by yairm210** [comment]
Making the population not work doesn't reduce the population size, doesn't reduce the amount of food required, and doesn't affect happiness - maybe you're thinking of "raze"
In any case, working is always better than not working, so unassigned population *should* always be reassigned. More stats is *always* positive

**Comment by yairm210** [comment]
Starving is not achieved by "not working"... That's what "raze" is for

**Comment by yairm210** [comment]
Closing as 'not a bug'

**Comment by adeverteuil** [comment]
Hi @yairm210, I searched for this issue as well. I would also want citizens to remain unassigned when the citizen manager is set to Manual mode. Raze is not what I'm looking for. I want the citizens to provide 1 production, and slowly decrease the population over time using food deficit. I conquered a city that has 16 population but almost no production tiles. I want to control population by reassigning them from useless food tiles to unassigned (for production).
Thanks.
